tools/bismark/bismark_bowtie_wrapper.xml

```diff
@@ -1,582 +0,0 @@
-<tool id="bismark_bowtie" name="Bismark" version="0.14.3">
```

Comment:
No legit reason to be honest, just that no one in the lab of epigenetics with whom I'm working with would use bowtie 1 for the alignment... Sorry if I deleted it in my development dir and forget to put it back in the PR... but you could re-add it if you like.